.NET/Microsoft.Recognizers.Definitions/BasePhoneNumbers.cs

```diff
@@ -18,7 +18,11 @@ namespace Microsoft.Recognizers.Definitions
 	public static class BasePhoneNumbers
 	{
 		public const string NumberReplaceToken = "@builtin.phonenumber";
-		public static readonly Func<string, string, string> IntegerRegexDefinition = (placeholder, thousandsmark) => $@"(((?<!\d+\s*)-\s*)|((?<=\b)(?<!(\d+\.|\d+,))))\d{{1,3}}({thousandsmark}\d{{3}})+(?={placeholder})";
-		public const string PlaceHolderDefault = "\\D|\\b";
+		public const string NoLetterPrefixRegex = @"(?<![a-z|0-9])";
```

Comment:
Do we need to consider "A-Z"? If so, please add the related test case.

Reply:
Which cases actually match this? Letters are no valid as prefix, nor suffix.

Reply:
Also, can't we just restrict the boundaries of the regexes instead of making all regexes more complex by appending more patterns to their start and end?

Reply:
It's for getting rid of the invalid phone number, such as "s6666666", "6666666s" or something else. And I'm going to add the "A-Z".

Reply:
If so, then I think it should be [^0-9]

Reply:
@tellarin  mention that something like "\b"

Reply:
Yep. Like https://docs.microsoft.com/en-us/dotnet/standard/base-types/anchors-in-regular-expressions#word-boundary-b

Reply:
The '\b' does extract "49176 12345678" from "+49176 12345678" and "55) 3333-33333" from "(55) 3333-33333", and I think it's not suitble for this situation.